src/styles/blocks/container.scss

```diff
@@ -0,0 +1,14 @@
+.container {
+  margin: 0 auto;
```

Comment:
Don't use outer spacings (margins) for BEM blocks. Use mix of classes, because `.container` is BEM block